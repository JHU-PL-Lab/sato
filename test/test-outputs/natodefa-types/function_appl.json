[
  {
    "err_errors": [
      [
        "Error_match",
        {
          "err_match_aliases": [ "not_fn" ],
          "err_match_val": "2",
          "err_match_expected": "Function",
          "err_match_actual": "Integer"
        }
      ]
    ],
    "err_input_seq": [],
    "err_location": "not_fn  1",
    "err_steps": 178
  }
]