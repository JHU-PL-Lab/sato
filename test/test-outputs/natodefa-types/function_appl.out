Type errors for:
- Input sequence : []
- Found at expr  : not_fn  1
--------------------
* Value    : not_fn = 2
* Expected : Function
* Actual   : Integer
1 errors found. No further control flows exist.
